.github/workflows/Go-SDK-Check-k8s-integration.yaml

```diff
@@ -23,7 +23,7 @@ on:
     branches:
       - main
 env:
-  GO_VERSION: 1.19
+  GO_VERSION: ^1.20.2
```

Comment:
Inside function `unmarshalObject` define the `type Unmarshal *U` and works with go >= 1.20

Reply:
Does it works or *only* works with 1.20+?

Reply:
Exactly.

Version 1.19 the error occurs:
```
type declarations inside generic functions are not currently supported
```

Reply:
I see, wee need to be cautions on situations like these, because there could be users that would be using older go lang versions and can't update it.

@ricardozanini @lsytj0413 wdyt?

Reply:
Agree with @spolti ，go1.19 is still under active supported，so we should promise this sdk will working with go1.19

Reply:
Solved the support to go 1.19.